ui/page/security_tools_page.go

```diff
@@ -55,15 +55,15 @@ func (pg *SecurityToolsPage) OnNavigatedTo() {
 
 }
 
-// Layout draws the page UI components into the provided layout context
+// Layout draws the page UI components into the provided C
```

Comment:
i think the previous comment is still valid and more explanatory, for all instance of this